backend/api/query.py

```
@@ -0,0 +1,519 @@
+        reddit_calls = 0
+        all_results = []
+        
+        if request.usernames:
+            # Get specific user profiles
+            for username in request.usernames:
+                reddit_calls += 1
+                try:
+                    user_data = collector.reddit_client.get_user_info(username)
+                    if user_data:
+                        all_results.append(user_data)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)